src/01-functions.js

```diff
@@ -168,7 +191,9 @@ function joinStringsWithSpaces() {}
  *  getFirstAndLastCharacter("upwards");
  *  //> "us"
  */
-function getFirstAndLastCharacter() {}
+function getFirstAndLastCharacter(a) {
+  return a.charAt(0) + a.charAt(a.length-1)
```

Comment:
Nice ` charAt() ` call.  Remember we can index strings like arrays.  ` a.charAt(0) ` => `  a[0] `